apidef/oas/oas_test.go

```diff
@@ -215,8 +215,6 @@ func TestOAS_ExtractTo_ResetAPIDefinition(t *testing.T) {
 		"APIDefinition.VersionData.Versions[0].ExtendedPaths.TrackEndpoints[0].Method",
 		"APIDefinition.VersionData.Versions[0].ExtendedPaths.DoNotTrackEndpoints[0].Path",
 		"APIDefinition.VersionData.Versions[0].ExtendedPaths.DoNotTrackEndpoints[0].Method",
-		"APIDefinition.VersionData.Versions[0].ExtendedPaths.Internal[0].Path",
```

Comment:
can you please provide some context as of why are those lines removed? It's a bit unclear for me

Reply:
> 	// The expectedFields value lists fields that do not support migration.
	// When adding a migration for ExtendedPaths sections, clear the list of
	// fields below, and clear the value in ExtendedPaths.Clear() function.

assert.Equal(t, expectedFields, noOASSupportFields)

Once a field is extracted from OAS, it should be cleared in apidef, so as to not keep any previous values.